source/server/connection_handler_impl.cc

```diff
@@ -144,6 +159,25 @@ void ConnectionHandlerImpl::setListenerRejectFraction(UnitFloat reject_fraction)
   }
 }
 
+Network::InternalListenerCallbacksOptRef
+ConnectionHandlerImpl::findByAddress(const Network::Address::InstanceConstSharedPtr& address) {
+  auto listener_it =
```

Comment:
assert that the address type is internal?

Reply:
done